src/parser/flow_lexer.ml

```diff
@@ -724,65 +767,86 @@ let token (env: Lex_env.t) lexbuf : result =
       is_tail
     ))
 
-  | binnumber, (letter | '2'..'9'), Star alphanumeric ->
+  | binbigint ->
```

Comment:
I mentioned it in the top level review, but this case needs to come after the `(* Numbers cannot be immediately followed by words *)` case. You need to do that here and everywhere else.

Reply:
Wouldn't it break because `(* Numbers cannot be immediately followed by words *)` case would match `n` character at the end earlier if I move this to top?